apps/costcontrol/js/common.js

```diff
@@ -71,6 +71,23 @@ var Common = {
     'app://search.gaiamobile.org/manifest.webapp'
   ],
 
+  SYSTEM_MANIFEST: 'app://system.gaiamobile.org/manifest.webapp',
+
+  BROWSER_APP: {
+    manifestURL: 'app://browser.gaiamobile.org/manifest.webapp',
+    origin: '',
+    manifest: {
+      'icons': {
```

Comment:
Could you write `icons` instead of the string `'icons'` or is it a reserved word?